### PR TITLE
Raft host/port rename

### PR DESCRIPTION
**Pull request by jumaffre**
As we now use the node-to-node connections for more than just raft (i.e. PBFT, forwarding), the CLI options and documentation are no longer raft-specific.

### CODE REVIEW

**Comment by jumaffre** [comment]
> I wonder if —consensus-address host:port and —rpc-address host:port would be clearer instead?

@achamayou `--rpc-address` sounds better indeed. Consensus is a little bit specific perhaps as we also use this connection for forwarding - I prefer `node-address` or `peer-address` and clearly explain what it is in the `--help` of `cchost`. I'll combine `-host` and `-port` into `-address` too.

**Comment by olgavrou** [comment]
I think I prefer `node` to `peer` since we use the former quite a bit but not the latter really.

File: sphinx/source/recovery.rst [hunk 1]
```diff
@@ -22,9 +22,8 @@ To initiate the first phase of the recovery protocol, one of several nodes must
 
 .. code-block:: bash
 
-    $ cchost --start=recover --enclave-file=/path/to/application --raft-host=raft_ip --raft-port=raft_port
-    --tls-host=tls_ip --tls-pubhost=tls_public_ip --tls-port=tls_port --ledger-file=ledger_file
-    --node-cert-file=/path/to/node_certificate --quote-file=/path/to/quote
+    $ cchost --start=recover --enclave-file=/path/to/application --node-address=node_ip:node_port --rpc-address=tls_ip:tls_port
```
Comment (achamayou) [inline sphinx/source/recovery.rst]: host:port, or perhaps rpc_host:rpc_port? Seems odd to keep the tls_ name here.

File: sphinx/source/recovery.rst [hunk 2]
```diff
@@ -22,9 +22,8 @@ To initiate the first phase of the recovery protocol, one of several nodes must
...
+    --ledger-file=ledger_file --node-cert-file=/path/to/node_certificate --quote-file=/path/to/quote
```
Comment (achamayou) [inline sphinx/source/recovery.rst]: Interestingly, I think we will be able to get rid of those last two arguments in the new light genesis startup mode.

File: sphinx/source/recovery.rst [hunk 3]
```diff
@@ -65,13 +64,13 @@ Once the public crash-fault tolerant network is established, members are allowed
 
 .. code-block:: bash
 
-    $ memberclient accept_recovery --sealed-secrets=/path/to/sealed/secrets/file --cert=/path/to/member1/cert --privk=/path/to/member1/private/key --host=leader_ip --port=leader_port --ca=/path/to/new/network/cert
+    $ memberclient accept_recovery --sealed-secrets=/path/to/sealed/secrets/file --cert=/path/to/member1/cert --privk=/path/to/member1/private/key --server-address=leader_rpc_ip:leader_port_ip --ca=/path/to/new/network/cert
```
Comment (achamayou) [inline sphinx/source/recovery.rst]: leader_rpc_port?

File: sphinx/source/start_network.rst [hunk 4]
```diff
@@ -26,22 +28,22 @@ Once the initial set of nodes is running, the ``nodes.json`` file specifying the
     $ cat nodes.json
     [
         {
-            "pubhost": "tls_public_ip0",
+            "pubhost": "rpc_public_ip0",
             "cert": [<output node0 cert bytes>],
-            "host": "raft/tls_ip0",
+            "host": "raft/rpc_ip0",
```
Comment (achamayou) [inline sphinx/source/start_network.rst]: shouldn't say raft anymore?

File: src/ds/cli_helper.h [hunk 5]
```diff
@@ -0,0 +1,66 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the Apache 2.0 License.
+
+#include <CLI11/CLI11.hpp>
+
+namespace cli
+{
+  static constexpr auto DEFAULT_ADDR = "0.0.0.0:4567";
```
Comment (achamayou) [inline src/ds/cli_helper.h]: Do we ever need this default?

Comment (jumaffre) [reply src/ds/cli_helper.h]: Good point. I have removed it.

**Pull request merged**